bin/cohttp_server_lwt.ml

```diff
@@ -147,33 +147,52 @@ let start_server docroot port host index verbose cert key () =
   in
   Server.create ~mode config
 
-let host = ref "0.0.0.0"
-let port = ref 8080
-let index = ref "index.html"
-let verbose = ref false
-let ssl_cert = ref None
-let ssl_key = ref None
-let rest = ref []
-
-let usage = "usage " ^ Sys.argv.(0) ^ " [DOCROOT]"
-
-let arglist = [
-  ("-p", Arg.Int (fun i -> port := i), ": TCP port to listen on");
-  ("-s", Arg.String (fun s -> host := s), ": IP address to listen on");
-  ("-i", Arg.String (fun s -> index := s), ": Name of index file in directory");
-  ("-v", Arg.Bool (fun b -> verbose := b), ": logging output to console");
-  ("-c", Arg.String (fun s -> ssl_cert := Some s), ": certificate file");
-  ("-k", Arg.String (fun s -> ssl_key := Some s), ": key file");
-]
-
-let _ =
-  try Arg.parse arglist (fun x -> rest := x :: !rest) usage;
-    let dir =
-      match !rest with
-      | [] -> "."
-      | dir::_ -> dir in
-    Lwt_main.run (
-      start_server dir !port !host !index !verbose !ssl_cert !ssl_key ())
-  with
-  | Failure s -> print_endline s
-  | Sys_error s -> print_endline s
+let lwt_start_server docroot port host index verbose cert key =
+  Lwt_main.run (start_server docroot port host index verbose cert key ())
+
+open Cmdliner
+
+let host = 
+  let doc = "IP address to listen on" in
+  Arg.(value & opt string "0.0.0.0" & info ["s"] ~docv:"HOST" ~doc)
+
+let port =
+  let doc = "TCP port to listen on" in
+  Arg.(value & opt int 8080 & info ["p"] ~docv:"PORT" ~doc)
+
+let index =
+  let doc = "Name of index file in directory" in
+  Arg.(value & opt string "index.html" & info ["i"] ~docv:"INDEX" ~doc)
+
+let verb =
+  let doc = "logging output to console" in
+  Arg.(value & flag & info ["v"; "verbose"] ~doc)
```

Comment:
Yeah, I should add a dot. I used [this program](https://github.com/mirage/ocaml-cohttp/blob/master/bin/cohttp_curl_lwt.ml#L67) as my reference. It uses uppercase instead. I am not sure now